src/App.js

```diff
@@ -1,13 +1,51 @@
 import React, { Component } from 'react'
-import { Instructions } from './components'
+import { Instructions, Counter } from './components'
 import './styles/app.css'
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { userInput: '', initialCount: 0 }
+
+    this.handleChange = this.handleChange.bind(this)
```

Comment:
Instead of doing this, ES6 (or ES7) has a nice thing with auto binding whereby, in the class, you write your functions like this:
```javascript
handleChange = event => {
  ...
}
```

instead of
```javascript
handleChange(event) {
  ...
}
```

and then you don't have to manually `bind` them in the constructor 🎉